pkg/reconciler/trigger/trigger.go

```diff
@@ -236,6 +252,55 @@ func (r *Reconciler) reconcileRetryTopicAndSubscription(ctx context.Context, tri
 	return nil
 }
 
+// getPubsubRetryPolicy gets the eventing retry policy from the Broker delivery
+// spec and translates it to a pubsub retry policy.
+func getPubsubRetryPolicy(spec *eventingduckv1beta1.DeliverySpec) *pubsub.RetryPolicy {
+	var backoffDelay *string
+	var backoffPolicy *eventingduckv1beta1.BackoffPolicyType
+	if spec == nil {
+		backoffDelay = &brokerv1beta1.DefaultBackoffDelay
+		backoffPolicy = &brokerv1beta1.DefaultBackoffPolicy
```

Comment:
From the defaulting logic, shouldn't the spec always present?

Reply:
I'm not sure it is guaranteed if the webhook fails for some reason.

Reply:
In case of webhook not available, the requests would fail.
The code is fine if you prefer the current way.